### DISCUSSION TITLE
Possible to enable JIT?

### DISCUSSION THREAD

**Discussion by robertgdev** [NativePHP/laravel#455] [Q&A]
Just a quick question: is it possible to enable JIT in the resulting app? Asking because I want to write something that involves lots of looping/repeating math, and without JIT enabled, PHP would probably be too slow.

**Comment by simonhamp** [comment]
Although I haven't tried it yet, to my understanding, yes it should be possible. Opcache has been compiled in to the static binaries for a little while now.

It should just be a case of [passing the appropriate ini settings](https://stitcher.io/blog/php-8-jit-setup) at runtime, which you should [already be able to do](https://nativephp.com/docs/1/getting-started/configuration#customize-phpini).

I'd love to hear how you get on!

**Comment by robertgdev** [comment]
Awesome, thank you!
